catalog/models.py

```diff
@@ -38,4 +39,201 @@ def __str__(self) -> str:
         Returns:
             str: model data identifier.
         """
-        return self.name or self.identifier
\ No newline at end of file
+        return self.name or self.identifier
+
+
+class Catalogs (models.Model):
```

Comment:
Remove whitespace between class name and `(`

Reply:
Removed.